Lib/typing.py

```diff
@@ -1359,7 +1359,7 @@ def __ror__(self, left):
 
     @_tp_cache
     def __getitem__(self, args):
-        # Parameterizes an already-parameterized object.
+        # Parametrizes an already-parameterized object.
```

Comment:
I don't know which of "parameterize" and "parametrize" is correct, I think we may have discussed it in some earlier PR. But the two occurrences of the verb in this line should stay consistent. And if both spellings are acceptable, we should just stick with what we already have.

Reply:
Both are apparently correct. I'll revert this part.